### PR TITLE
backup guest fix

### PR DESCRIPTION
**Pull request by wanyaoqi**
**这个 PR 实现什么功能/修复什么问题**:
- fix code of create backup guest 
- scheduler support host group by wire

**是否需要 backport 到之前的 release 分支**:
NONE
/area region scheduler host
/cc @swordqiu @zexi

### CODE REVIEW

File: pkg/scheduler/api/sched.go [hunk 1]
```diff
@@ -31,10 +31,11 @@ import (
...
 
-	Tag         string   `json:"tag"`
-	Type        string   `json:"type"`
-	IsContainer bool     `json:"is_container"`
-	Candidates  []string `json:"candidates"`
+	Tag                string   `json:"tag"`
+	Type               string   `json:"type"`
+	IsContainer        bool     `json:"is_container"`
+	PreferCandidates   []string `json:"candidates"`
+	RequiredCandidates int      `json: "required_candidates"`
```
Comment (swordqiu) [inline pkg/scheduler/api/sched.go]: `json: "required_candidates"` 去掉空格

Comment (wanyaoqi) [reply pkg/scheduler/api/sched.go]: fixed

**Pull request merged**